pkg/operator/staticpod/controller/prune/prune_controller.go

```diff
@@ -78,12 +71,12 @@ func NewPruneController(
 
 		prunerPodImageFn: getPrunerPodImageFromEnv,
 	}
-	c.ownerRefsFn = c.setOwnerRefs
+	c.retrieveStatusConfigMapOwnerRefsFn = c.createStatusConfigMapOwnerRefs
 
 	return factory.New().WithInformers(operatorClient.Informer()).WithSync(c.sync).ToController("PruneController", eventRecorder)
 }
 
-func getRevisionLimits(operatorSpec *operatorv1.StaticPodOperatorSpec) (int32, int32) {
+func revisionsToKeep(operatorSpec *operatorv1.StaticPodOperatorSpec) (int, int) {
 	failedRevisionLimit := defaultRevisionLimit
 	succeededRevisionLimit := defaultRevisionLimit
 	if operatorSpec.FailedRevisionLimit != 0 {
```

Comment:
why not `operatorSpec.FailedRevisionLimit > 0` ?

Reply:
done